### ISSUE TITLE
On this page list should set the focus to that section

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#5540]
## ❌ Current behaviour (issue)
When a link on the _On this page_ list is selected the page scrolls to that section but the focus remains in the list.

### How to reproduce
1. Browse to the profile page
2. Use tab navigation to select a link from the _On this page_ list
3. Observe that the page scrolled to that section
4. Continue tabbing through the page until focus gets to the desired section

## ✔ Expected behaviour
Selecting the _On this page_ link should set the focus immediately to that section

### Acceptance Criteria
- [ ] Selecting the _On this page_ link should set the focus immediately to that section

## Additional information

https://images.zenhubusercontent.com/364642496/b2e300c7-282d-41c7-a11b-14600f0cd3c9/simplescreenrecorder_2023_01_31_09_04_41.mp4

Suggestion from Eric:
This shouldn't be too difficult either. We have [a ref to the HTML element](https://github.com/GCTC-NTGC/gc-digital-talent/blob/02ae1e85d7048d96322f6f2f161cd9beef640b2b/frontend/common/src/components/Link/ScrollToLink.tsx#L17-L24), should be as simple as `targetSection.focus();`
